src/sentry/api/endpoints/organization_release_assemble.py

```diff
@@ -57,7 +58,19 @@ def post(self, request: Request, organization, version) -> Response:
         checksum = data.get("checksum", None)
         chunks = data.get("chunks", [])
 
-        state, detail = get_assemble_status(AssembleTask.RELEASE_BUNDLE, organization.id, checksum)
+        upload_as_artifact_bundle = False
+        project_ids = []
+        if features.has("organizations:sourcemaps-upload-release-as-artifact-bundle", organization):
+            upload_as_artifact_bundle
+            project_ids = [project.id for project in release.projects.all()]
```

Comment:
Is this done because `ReleaseFile` has an implicit relation to the projects connected to a release?

Reply:
Or rather because the upload endpoint is per-org, and not per-project. The `ReleaseFile` only has a direct org relation, and an indirect project relation via the release, yes.